rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -204,14 +204,49 @@ else
       "[INFO]" "begin to download ssh key pair from hdfs ..."
   hdfs dfs -get "${hdfs_ssh_folder}/.ssh/" "/root/" || exit 1
 fi
-# Start ssh service
-start_ssh_service
+chmod 400 ~/.ssh/$APP_ID
 # Generate ssh connect info file in "PAI_CONTAINER_ID-PAI_CURRENT_CONTAINER_IP-PAI_CONTAINER_SSH_PORT" format on hdfs
 hdfs dfs -touchz ${hdfs_ssh_folder}/$PAI_CONTAINER_ID-$PAI_CONTAINER_HOST_IP-$PAI_CONTAINER_SSH_PORT || exit 1
 
+# Generate ssh config
+ssh_config_path=${HDFS_LAUNCHER_PREFIX}/${PAI_USER_NAME}/${PAI_JOB_NAME}/ssh/config
```

Comment:
To avoid the confusion with the config of CloudDocker and increase the ease of code maintenance, I suggest use another directory, say `${HDFS_LAUNCHER_PREFIX}/${PAI_USER_NAME}/${PAI_JOB_NAME}/mpi`.